dev-lang/solidity/solidity-0.4.24.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils
+
+MY_P="${PN}_${PV}"
+
+DESCRIPTION="The Solidity Contract-Oriented Programming Language"
+HOMEPAGE="https://github.com/ethereum/solidity"
+SRC_URI="https://github.com/ethereum/${PN}/releases/download/v${PV}/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
```

Comment:
https://github.com/ethereum/solidity/blob/develop/libsolc/libsolc.cpp says GPL-3+. Any reason to restrict?